apps/explorer/src/utils/static/mock_data.json

```diff
@@ -595,7 +595,7 @@
             "readonly": "true",
             "objType": "0x2::coin::Coin<0x2::USD::USD>",
             "data": {
-                "tx_digest": "Da4vHc9IwbvOYblE8LnrVsqXwryt2Kmms+xnJ7Zx5E4=",
```

Comment:
mock_data will be deprecated in favor of localnet e2e test, still added here for consistency